grid2op/tests/test_MultiMix.py

```diff
@@ -228,5 +229,41 @@ def test_forecast_toggle(self):
         assert isinstance(info, dict)
         assert done is not True
 
+    def test_bracket_access_by_name(self):
+        mme = MultiMixEnvironment(PATH_DATA_MULTIMIX)
```

Comment:
Here it would be nice to add the test that check:
- that the name of each mix are indeed different
- that if we pass a backend in the grid, everything is perfectly working, for now i think with the current implementation, both env have the same backend, which is terrible :-/